test/Dialect/FIRRTL/lowerXMR-error.mlir

```diff
@@ -0,0 +1,59 @@
+// RUN: circt-opt %s  --firrtl-lower-xmr -split-input-file -verify-diagnostics
```

Comment:
Micro-nit: I think we usually name these "-errors" ?

Reply:
Right, renamed it.